dev/AppNotifications/AppNotificationBuilder/AppNotificationTextProperties.h

```diff
@@ -34,6 +34,8 @@ namespace winrt::Microsoft::Windows::AppNotifications::Builder::implementation
         int m_maxLines{ 0 };
         winrt::hstring m_language{};
         bool m_useCallScenarioAlign{};
+
+        NotificationTelemetryHelper m_telemetryHelper{};
```

Comment:
Same concern. 

In essence, we should figure out a way to use a global rather than re-initializing this over and over again.

Also the implementation of helper should cache the values of AppId and IsPackaged to avoid expensive RPC calls to state repo.